packages/blade/.babelrc.js

```diff
@@ -20,7 +20,10 @@ module.exports = {
       ],
     },
     production: {
-      presets: ['@babel/preset-typescript', 'module:metro-react-native-babel-preset'],
+      presets: [
+        '@babel/preset-typescript',
+        ['module:metro-react-native-babel-preset', { disableImportExportTransform: true }],
```

Comment:
Why `disableImportExportTransform`?

Reply:
because else by default this preset transforms exports to commonjs. It's the equivalent of [this](https://babeljs.io/docs/en/babel-preset-env#modules) but this preset accepts it through this flag. Not documented anywhere had to look into codebase 😢